src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/caller/CalledCopyRatioSegment.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.caller;
+
+import org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation.CopyRatioSegment;
+import org.broadinstitute.hellbender.utils.Utils;
+
+public class CalledCopyRatioSegment extends CopyRatioSegment {
+    public enum Call {
+        AMPLIFICATION_CALL("+"),
+        DELETION_CALL("-"),
+        NEUTRAL_CALL("0");
+
+        private final String outputString;
+
+        Call(final String outputString) {  this.outputString = outputString; }
+
+        public String getOutputString() {
+            return outputString;
+        }
+    }
+
+    private final Call call;
+
+    public CalledCopyRatioSegment(final CopyRatioSegment segment,
+                                  final Call call) {
+        super(segment.getInterval(), segment.getNumPoints(), segment.getMeanLog2CopyRatio());
+        this.call = Utils.nonNull(call);
+    }
+
+    public Call getCall() {
+        return call;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        if (!super.equals(o)) {
+            return false;
+        }
+        final CalledCopyRatioSegment that = (CalledCopyRatioSegment) o;
+        return call == that.call;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = super.hashCode();
+        result = 31 * result + call.hashCode();
```

Comment:
This would be equally clear as a one-liner: `return 31 * super.hashCode() + call.hashCode();`

Reply:
Same thing here.